src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/StripEmbeddedLibraries.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Mono.Cecil;
+using Mono.Linker;
+using Mono.Linker.Steps;
+using System;
+using System.Linq;
+using Xamarin.Android.Tasks;
+
+namespace MonoDroid.Tuner
+{
+	public class StripEmbeddedLibraries : BaseStep
+	{
+		protected override void ProcessAssembly (AssemblyDefinition assembly)
+		{
+			if (!Annotations.HasAction (assembly))
+				return;
+
+			var action = Annotations.GetAction (assembly);
+			if (action == AssemblyAction.Skip || action == AssemblyAction.Delete)
+				return;
```

Comment:
I think we should only skip the ones with `AssemblyAction.Delete` here. The `AssemblyAction.Skip` means skip the linking for that assembly, while we might still want to strip it.

Reply:
Wanting to make sure I understand, if I look at the `OutputStep` here: https://github.com/mono/linker/blob/c85962b3aa514ff23161f0b5a1ef16fa89b9e844/linker/Linker.Steps/OutputStep.cs#L118-L137

It seems like it isn't going to output `Skip` assemblies at all. So we shouldn't worry about stripping them? They look like they aren't getting saved/copied to the destination directory?

Reply:
@radekdoulik The good news is @jonathanpeppers is pretty good with Tests as well :)  We probably have a test which has embedded library/zip already. 

The end goal is the final assemblies in the apk (or fastdev dir) do not contain the zips or the environment files.

Reply:
I'll make sure we have a test with `--linkskip` that checks if the zips are missing

Reply:
Nice! Only missing thing here is to test that the assembly wasn't linked. That can be done by adding a method which is not used in the app and check that it remained there after the linking.